lib/ruby_lsp/ruby_lsp_rails/definition.rb

```diff
@@ -0,0 +1,94 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Rails
+    # ![Definition demo](../../definition.gif)
```

Comment:
Let's change this documentation to mention only the feature contributed by `ruby-lsp-rails` (go to definition for callbacks).